package.json

```diff
@@ -897,6 +897,18 @@
                 {
                     "command": "aws.resources.viewDocs",
                     "when": "false"
+                },
+                {
+                    "command": "aws.ecs.runCommandInContainer",
+                    "when": "false"
+                },
+                {
+                    "command": "aws.ecs.enableEcsExec",
+                    "when": "false"
```

Comment:
when do these get enabled?

Reply:
Never. These are for the command palette, so they'll never show up.

This could be another entry-point but we need to prompt for which region/cluster to use. Though if we did this for every command we'd have _a lot_ of commands. Plus the text would need to be different. Crazy idea: expose the entire tree structure as a quick pick wizard with commands to execute available as options.